src/view/components/semantic/modules/dropdown/dropdown.ts

```diff
@@ -240,8 +241,13 @@ export class Dropdown implements AfterViewInit, OnDestroy, ControlValueAccessor
    */
   private _applyArrowNavFix($dropdown) {
     let $searchField = $dropdown.children('input.search')
+    let enterEvent = this.enter;
     $searchField.on('keyup', (event:any)=> {
       if (DO_NOT_SEARCH_ON_THESE_KEY_EVENTS[event.keyCode]) {
+        if (event.keyCode == 13 && enterEvent){
```

Comment:
Don't test for things you know are always false. In this case, you declared the event emitter in the constructor -- it will never be null. If it WERE to become null, we'd rather find out about it from the NPE than to wonder why this feature was silently failing.